### DISCUSSION TITLE
Learning Web3Auth

### DISCUSSION THREAD

**Discussion by caesarvdidsert** [Web3Auth/Web3Auth#1200] [Help]
What is the most effective way to learn how to implement Web3Auth without any previous coding experience?

Is it recommended to do buy a JavaScript Basic Knowledge Course, or is there something else I should try?

(in my case I'm looking to connect Auth0 with Web3Auth)

Thanks,

Caesar

**Comment by Saviour1001** [comment]
You should know javascript, you can build your application using the React Library for the Web3Auth and Auth0 apps. We have created an [example app](https://github.com/Web3Auth/examples/tree/main/web-core-sdk/custom-authentication/auth0-react-core-example) which already has Auth0 integration, have a look.
